Content.Server/StationEvents/StationEvent.cs

```diff
@@ -27,12 +33,47 @@ public abstract class StationEvent
         /// </summary>
         protected virtual string EndAnnouncement { get; } = null;
 
+        /// <summary>
+        /// Starting audio of the event.
+        /// </summary>
+        protected virtual string StartAudio { get; } = "/Audio/Effects/alert.ogg";
+
+        /// <summary>
+        /// Ending audio of the event.
+        /// </summary>
+        protected virtual string EndAudio { get; } = null;
+
+        /// <summary>
+        /// Can the false alarm fake this event?
+        /// </summary>
+        public virtual bool Fakeable { get; } = true;
```

Comment:
IMO just check the type of the event in the fakeable event rather than hardcoding in fakeable stuff given it'll effectively be the same (at least with the current events).

Reply:
understandable though we have to return this if we are going to have wizard-like events and Halloween/Christmas event things.

Reply:
Do you mean you'd change it for seasonal events? Still seems better to leave all of that to the random event IMO.